var/spack/repos/builtin/packages/ffb/package.py

```diff
@@ -150,12 +150,14 @@ def edit(self, spec, prefix):
         cxx_fortran_flags = []
         if spec.satisfies('%gcc'):
             cxx_fortran_flags.append('-lgfortran')
-        elif spec.satisfies('%intel'):
-            cxx_fortran_flags.expand(['-lifcore', '-limf'])
+            m = FileFilter(editfile)
+            m.filter('-lifcore -limf', ' '.join(cxx_fortran_flags))
         elif spec.satisfies('%fj'):
             cxx_fortran_flags.append('--linkfortran')
-        m = FileFilter(editfile)
-        m.filter('-lifcore -limf', ' '.join(cxx_fortran_flags))
+            m = FileFilter(editfile)
+            m.filter('-lifcore -limf', ' '.join(cxx_fortran_flags))
+        elif spec.satisfies('%intel'):
```

Comment:
did you intend to remove the flags with the intel compiler?

Reply:
The original FILES was set up for Intel, so we removed the Intel flags as they are not needed.